opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -95,6 +93,28 @@ public synchronized void reset() {
         return new ArrayList<>(this.finishedSpans);
     }
 
+    /**
+     * @return a copy of all finish()ed Traces(Spans) started by this MockTracer grouped by traceId and spanId in HashMap format.
```

Comment:
It does not return a copy

Reply:
Thanks for pointing out, I just copied so words from the comments of `finishedSpans` and didn't realised it was wrong.